### DISCUSSION TITLE
Actividad Docker - Grupo 14

### DISCUSSION THREAD

**Discussion by Lele3002** [frlp-utn-ingsoft/catedra#115] [Show and tell]
https://github.com/JuanbyGuada/vetsoft

**Comment by rodriguezemautn** [comment]
Correcciones: 
- archivo se llama “env-example” no lleva extencion, luego se edita y se lo renombra .env
- no es de buenas practicas colocar variables de entorno sensibles en el env-example
- no despliega el contenedor con modo detach (-d) al cerrar la consola se detiene el contenedor
 
Puntos Aprobados:
* construye la imagen
* buen tamaño de imagen
* despliega el contenedor
* buen trabajo con la GUI

Salida de comandos:
$ docker images -a
vetsoft-app   2.0       b855c10beb4d   2 minutes ago   308MB
$ docker ps -a
452508f5f88e   vetsoft-app:2.0   "python manage.py ru…"   About a minute ago   Exited (0) 45 seconds ago             lucid_buck

**Reply by JuanbyGuada** [reply]
Buenas tardes! Ya aplicamos las correcciones. Saludos!
